content/docs/other-resources.md

```diff
@@ -3,30 +3,34 @@ title: Other Resources
 weight: 160
 ---
 
-
 ## Notepad Replacement
+
 Notepad is a default text editor shipped with Windows. You may want to use Notepad++ instead of Notepad. However, there's no obvious way to do it.
-From the version 7.5.9 onward, you can run the following command to make Notepad++ replace Notepad (run in `cmd.exe` with Administrator privileges)[†](#registry-edit-warning):
+From the version 7.5.9 onward, you can run the following command to make Notepad++ replace Notepad (run in `cmd.exe` with Administrator privileges[†](#registry-edit-warning)):
```

Comment:
The footnote applies to the whole sentence, not just the parenthetical, which is why it was outside the parentheses.  Do you have a reason for changing this?

Reply:
I though it is more connected with what is in parenthesis, but you are right, it ifor the whole sentence, reverted.